runtime/bcutil/ComparingCursor.cpp

```diff
@@ -541,7 +541,8 @@ ComparingCursor::isRangeValidForUTF8Ptr(J9UTF8 *utf8)
 	U_8 *ptr = (U_8*)utf8;
 
 	if (_checkRangeInSharedCache) {
-		return FALSE != j9shr_Query_IsAddressInCache(_javaVM, utf8, J9UTF8_TOTAL_SIZE(utf8));
+		return FALSE != (j9shr_Query_IsAddressInCache(_javaVM, utf8, sizeof(J9UTF8))
```

Comment:
Pls remove the redundant `FALSE !=`

Reply:
and the redundant brackets.

Reply:
Hmm, nm I guess this is needed to convert a C boolean into a C++ boolean.

Reply:
I think with the `&&` we should be fine. Ill try it on windows to verify.